client/my-sites/plans/jetpack-plans/constants.ts

```diff
@@ -156,6 +159,64 @@ export const EXTERNAL_PRODUCTS_SLUG_MAP: Record< string, () => SelectorProduct >
 	[ PRODUCT_JETPACK_CRM_MONTHLY ]: EXTERNAL_PRODUCT_CRM_MONTHLY,
 };
 
+// Jetpack Stats
+
+const STATS_COMMERCIAL_PRICE = 10;
+const STATS_COMMERCIAL_CURRENCY = 'USD';
+
+export const INDIRECT_CHECKOUT_PRODUCT_STATS = (): SelectorProduct => ( {
+	productSlug: PRODUCT_JETPACK_STATS_MONTHLY,
+	term: TERM_MONTHLY,
+	displayTerm: TERM_ANNUALLY,
+	type: ITEM_TYPE_PRODUCT,
+	costProductSlug: PRODUCT_JETPACK_STATS_MONTHLY,
+	monthlyProductSlug: PRODUCT_JETPACK_STATS_MONTHLY,
+	iconSlug: 'jetpack_stats',
+	displayName: translate( 'Stats' ),
+	shortName: translate( 'Stats' ),
+	tagline: translate( 'Simple, yet powerful analytics' ),
+	displayPrice: STATS_COMMERCIAL_PRICE,
+	displayCurrency: STATS_COMMERCIAL_CURRENCY,
+	description: translate(
+		'With Jetpack Stats, you don’t need to be a data scientist to see how your site is performing.'
+	),
+	shortDescription: translate( 'Simple, yet powerful stats to grow your site.' ),
+	buttonLabel: translate( 'Get Stats' ),
+	features: {
+		items: [],
+	},
+	hidePrice: true,
+	// TODO: Update the URL when the Paid Stats purchase page is ready.
+	externalUrl: '/stats/purchase/{siteSlug}?flags=stats/paid-stats',
```

Comment:
```suggestion
	externalUrl: '/stats/purchase/{siteSlug}?from=calypso-plans&flags=stats/paid-stats',
```
Could we please add a `from` param here for analytics purposes.